main/main.cpp

```diff
@@ -2491,7 +2491,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		}
 	}
 
-	GLOBAL_DEF("internationalization/locale/include_text_server_data", false);
+	GLOBAL_DEF("internationalization/locale/include_text_server_data", true);
```

Comment:
As pointed out by Calinou, the size impact might be quite prohibitive on some platforms like Web or mobile.

Maybe as a first step we should keep it false, but at least make it `GLOBAL_DEF_BASIC`, and document clearly that it needs to be enabled when including translations for one of the relevant locales.

Reply:
Changed it back to `false` and to `GLOBAL_DEF_BASIC`.